test/Abp.Zero.SampleApp.Tests/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -577,6 +578,143 @@ public void Should_Write_History_But_Not_For_Property_If_Disabled_History_Tracki
             _entityHistoryStore.Received().Save(Arg.Is<EntityChangeSet>(s => predicate(s)));
         }
 
+        [Fact]
+        public void Should_Write_History_For_TPH_Tracked_Entities_With_One_To_Many_Relationship_Changes()
+        {
+            var studentId = CreateStudentAndGetId();
```

Comment:
not sure if we really need this helper method since we always fetch the student object via student id.

we can just create the student object in the test case then add lecture note to the created student object.

Reply:
by right we should be calling `ClearReceivedCalls` on `_entityHistoryStore` to avoid student creation entity changes being captured in this test case.